securedrop/models.py

```diff
@@ -641,12 +642,18 @@ def login(cls,
               username: str,
               password: str,
               token: str) -> 'Journalist':
+
         try:
             user = Journalist.query.filter_by(username=username).one()
         except NoResultFound:
             raise InvalidUsernameException(
                 "invalid username '{}'".format(username))
 
+        if user.username in Journalist.INVALID_USERNAMES and \
```

Comment:
The idea was to stop the creation of invalid user names, but we should not block log in if someone already created such username. @redshiftzero mentioned this in https://github.com/freedomofpress/securedrop/issues/5232#issuecomment-624740370

Reply:
@kushaldas Yes. That is why I also checked the value of `user.uuid`. Log in would not be interrupted if a user already has a invalid username. It will only be interrupted if both `user.username` and `user.uuid` is invalid.